testing/scenario_app/android/app/src/test/java/dev/flutter/scenarios/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package dev.flutter.scenarios;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
Would it ever be appropriate for developers to write unit tests here, now that we have official JUnit and Robolectric support? If not, then I would recommend deleting this example to avoid any confusion. Same for any other auto-generated stuff that we would expect developers to extend.

Reply:
This one doesn't make sense, will remove